config/webpack/production.js

```diff
@@ -0,0 +1,46 @@
+// Note: You must restart bin/webpack-dev-server for changes to take effect
+
+/* eslint global-require: 0 */
+
+const webpack = require('webpack')
+const merge = require('webpack-merge')
+const CompressionPlugin = require('compression-webpack-plugin')
+const BabiliPlugin = require('babili-webpack-plugin')
```

Comment:
[This article](https://blog.craftlab.hu/how-to-do-proper-tree-shaking-in-webpack-2-e27852af8b21) claimed that babili was preferable to uglify because it could understand the new syntaxes better.